### DISCUSSION TITLE
Add DETAIL to migration error output

### DISCUSSION THREAD

**Discussion by snewcomer** [sequelize/umzug#560] [General] [answered]
Currently an error logs the message property of the "ERROR".

```
    console.error(`${_cliColor2.default.red('ERROR:')} ${message}`);
```

In the case of an error, you get something like

```
SequelizeForeignKeyConstraintError: insert or update on table "my_table" violates foreign key constraint "...."
```

We could log the "DETAIL" similar to Postgres `log_error_verbosity` configuration.

```
    console.error(`${_cliColor2.default.red('DETAIL:')} ${error.original.detail}`);
```

This would greatly help with debugging migration issues with sequelize.

**Comment by mmkal** [comment, marked as answer]
Hmm I don't recognise that logging code, is it part of umzug?

**Reply by snewcomer** [reply]
Sorry @mmkal.  I errantly put this in the wrong place.  Sorry!  Can close this discussion.

https://github.com/sequelize/cli/blob/54ec29d65727873f88d0daa50e5e92ef7d00a4b8/src/helpers/view-helper.js#L38

**Reply by WikiRik** [reply]
Feel free to open an issue in the CLI repo for this :)
